packages/wdio-firefox-profile-service/src/launcher.js

```diff
@@ -10,7 +10,11 @@ export default class FirefoxProfileLauncher {
             return
         }
 
-        this.profile = new Profile()
+        if(this.config.firefoxProfile.profileDirectory) {
```

Comment:
What happens if this directory doesn't exist?

Reply:
The copy will return a default profile, so basically it works like you didn't configure it at all. Should I check it and throw an error?

Reply:
nah , I think this is fine